dired.sublime-settings

```diff
@@ -33,6 +33,13 @@
   //     "git_path": ""
   "git_path": "git",
 
+  // Path to hg executable.
+  // Wildcards are supported, e.g. if you use TortoiseHg it would be:
+  //     "hg_path": "%ProgramFiles%/TortoiseHg/hg.exe"
```

Comment:
1. It won’t work, because we don’t expand %var%, 

    ```python
    glob.glob(os.path.expandvars(os.path.expanduser(hg)))
    ```

2. In readme should be VCS instead of [CVS](https://en.wikipedia.org/wiki/Concurrent_Versions_System).

3. Please, run in your branch (it should remove old and merge commits, since they are pointless):

    ```
    git fetch upstream
    git rebase upstream/master
    git rebase --skip
    ```
    Perhaps, you’ll need replace `upstream` with `aziz`, look at output of `git remote -v` to be sure.